pkg/operator/ceph/controller/predicate.go

```diff
@@ -530,24 +205,23 @@ func WatchPredicateForNonCRDObject(owner runtime.Object, scheme *runtime.Scheme)
 					return true
 				}
 
-				// If the resource is a ConfigMap we don't reconcile
-				_, ok := e.ObjectNew.(*corev1.ConfigMap)
-				if ok {
+				// create a deep copy to evade the type constraint and allow use of .(type)
+				switch newObjCopy := e.ObjectNew.DeepCopyObject().(type) {
```

Comment:
It just occurred to me that I can probably cast this to `any()` and remove the pointless deep copy.

Reply:
Yep. That worked. ;)